samples/httpd_bsd/main.c

```diff
@@ -0,0 +1,107 @@
+#include "lwip/debug.h"
+#include "lwip/dhcp.h"
+#include "lwip/init.h"
+#include "lwip/netif.h"
+#include "lwip/sys.h"
+#include "lwip/tcpip.h"
+#include "lwip/timers.h"
+#include "netif/etharp.h"
+#include "pktdrv.h"
+#include <hal/video.h>
+#include <hal/input.h>
+#include <hal/xbox.h>
+#include <pbkit/pbkit.h>
+#include <xboxkrnl/xboxkrnl.h>
+#include <debug.h>
+
+#define USE_DHCP         1
+#define PKT_TMR_INTERVAL 5 /* ms */
+#define DEBUGGING        0
+
+#include "httpserver.h"
+
+struct netif nforce_netif, *g_pnetif;
+
+err_t nforceif_init(struct netif *netif);
+void http_server_netconn_init(void);
+static void packet_timer(void *arg);
+
+static void tcpip_init_done(void *arg)
+{
+  sys_sem_t *init_complete = arg;
```

Comment:
Since the indentation was changed from the httpd-sample anyway, use 4 spaces instead.